### PR TITLE
[ci-utils] Use google's apt repo to install gcloud and kubectl

### PR DESCRIPTION
**Pull request by chrisvittal**
See https://cloud.google.com/sdk/docs/install#deb 

### Security Assessment

- [x] This change has a low security impact

### CODE REVIEW

File: ci/Dockerfile.ci-utils [hunk 1]
```diff
@@ -1,12 +1,7 @@
...
-# source: https://cloud.google.com/storage/docs/gsutil_install#linux
-RUN curl --remote-name https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-447.0.0-linux-x86_64.tar.gz && \
-    tar -xf google-cloud-sdk-447.0.0-linux-x86_64.tar.gz && \
-    curl --remote-name https://dl.k8s.io/release/v1.21.14/bin/linux/amd64/kubectl && \
-    install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
-ENV PATH $PATH:/google-cloud-sdk/bin
+RUN hail-apt-get-install google-cloud-cli kubectl
 
 # https://bugs.mysql.com/bug.php?id=105288&thanks=sub
 RUN hail-apt-get-install xz-utils libncurses5 git && \
```
Comment (ehigham) [inline ci/Dockerfile.ci-utils:7]: Maybe combine these two to reduce the number of layers?

**Review by ehigham — CHANGES_REQUESTED** [review]

File: ci/Dockerfile.ci-utils (hunk 1 above)
Comment (chrisvittal) [reply ci/Dockerfile.ci-utils]: done

**Review by ehigham — APPROVED** [review]
nice

**Comment by chrisvittal** [comment]
Now just to figure out what broke VEP.

**Review by patrick-schultz — APPROVED** [review]
Much cleaner, thanks for figuring that out Chris!

**Pull request merged**